### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by praseodym**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    rm -rf /var/lib/apt/lists/*
+
+RUN mkdir -p /srv
+WORKDIR /srv
+
+COPY requirements.txt /srv
+RUN pip install --no-cache-dir -r requirements.txt
+COPY . /srv
+
+CMD ["python", "virtualenv/bin/gunicorn --workers=5 dienst2.wsgi"]
```
Comment (jgadelange) [inline Dockerfile]: gunicorn is standard installed in the python 3.5 docker unit?

Comment (praseodym) [inline Dockerfile]: It's in our requirements.txt :-)

Comment (jgadelange) [inline Dockerfile]: Can you link the line number? 😛

Comment (jgadelange) [inline Dockerfile]: I would say in deployment it is done by teamcity with the following line:
`pip install -U newrelic gunicorn`

Comment (praseodym) [inline Dockerfile]: 🙈 will fix

**Comment by praseodym** [comment]
Tested ✅

**Pull request merged**